conditions.md

```diff
@@ -36,7 +36,7 @@ This does not reflect a complete list of condition types, it is just to illustra
 
 | Condition Type | Description |
 | --- | --- |
-| `ReadyCondition` | All APIs have at least one `ReadyCondition` condition type that summarizes the overall operational state of the API object. Each CRD has a `IsReady()` method which returns a bool of the status of the instance/Ready condition https://github.com/openstack-k8s-operators/keystone-operator/blob/master/api/v1beta1/keystoneapi_types.go#L211 |
+| `ReadyCondition` | All APIs have at least one `ReadyCondition` condition type that summarizes the overall operational state of the API object. Each CRD has a `IsReady()` method which returns a bool of the status of the instance/Ready condition https://github.com/openstack-k8s-operators/keystone-operator/blob/main/api/v1beta1/keystoneapi_types.go#L211 |
```

Comment:
This one should probably be changed to point to a commit of some sort.  It no longer points to an `IsReady()` function and might change again even if we fix it but don't pin it to a commit.